tests/e2e/60-tracing.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { test, expect } from './rancher-test';
+import { Chart, RancherAppsPage } from './pages/rancher-apps.page';
+import { PolicyServersPage } from './pages/policyservers.page';
+
+/**
+ * Expect timeout has to be increased after telemetry installation on local cluster
+ *
+ */
+test('Install opentelemetry & jaeger', async ({ page }) => {
+  const apps = new RancherAppsPage(page)
+  const otelChart: Chart = { title: 'opentelemetry-operator', name: 'opentelemetry-operator', namespace: 'open-telemetry', version: '0.38.0', check: 'opentelemetry-operator' }
+  const jaegerChart: Chart = { title: 'Jaeger Operator', namespace: 'jaeger', check: "jaeger-operator" }
+
+  // Install OpenTelemetry
+  await apps.addRepository('open-telemetry', 'https://open-telemetry.github.io/opentelemetry-helm-charts')
+  await apps.installChart(otelChart)
+
+  // Install Jaeger
+  await apps.installChart(jaegerChart, d => {
+    d.jaeger.create = "true"
+    d.rbac.clusterRole = "true"
+  })
+});
+
+test('Enable tracing in Kubewarden', async ({ page, ui }) => {
+  const apps = new RancherAppsPage(page)
+  await apps.updateApp('rancher-kubewarden-controller', d => {
+    d.telemetry.enabled = true
```

Comment:
Should this be `d.telemetry.tracing.enabled` to respect the new chart? https://github.com/kubewarden/helm-charts/pull/331

Reply:
yes, I will change that, but we have to release new chart first